### PR TITLE
Add functionality to write to csv files

### PR DESCRIPTION
**Pull request by arsalanc-v2**
The following can now be done:
1. write lines to a file (csv, txt, etc)
2. check if any two files have the same content

### CODE REVIEW

**Comment by jjlee050** [comment]
@arsalanc-v2 Do you have a library found? coz this one need a library.

**Comment by arsalanc-v2** [comment]
> @arsalanc-v2 Do you have a library found? coz this one need a library.

I'm not planning on using a library

**Comment by arsalanc-v2** [comment]
> Wad abt xlsx?

excel can open csv files

**Comment by arsalanc-v2** [comment]
How should I rename the test methods (they're causing codacy to fail) ?

**Comment by jjlee050** [comment]
True. Leave it to u to explore but i might wanna consider more options to export data.

**Comment by jjlee050** [comment]
Can i ask oso why is this type epic. Epic is a big user story tht can be divided into smaller user stories.

**Comment by arsalanc-v2** [comment]
> True. Leave it to u to explore but i might wanna consider more options to export data.

Yeah I might look into it in the future once more important things are done

**Comment by arsalanc-v2** [comment]
> Can i ask oso why is this type epic. Epic is a big user story tht can be divided into smaller user stories.

Okay I misunderstood

File: src/test/java/seedu/address/model/ExportUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    @Test
+    public void writeLine_nullRow_throwsNullPointerException() throws IOException {
+        final File actual = folder.newFile("ActualExportUtilTest.csv");
+        thrown.expect(NullPointerException.class);
+        ExportUtil.writeLine(new FileWriter(actual), null);
+    }
+
+    @Test
```
Comment (iamjackslayer) [inline src/test/java/seedu/address/model/ExportUtilTest.java]: For test cases from lines 56 to 75, you might want to indicate from the method names that the file contents between actual and expected are different. For example writeLine_identicalLengthDifferentContent_returnsFalse().

File: src/test/java/seedu/address/model/ExportUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    public void writeLines_identicalContent_returnsTrue() throws IOException {
+        final File actual = folder.newFile("ActualExportUtilTest.csv");
+        ArrayList<String> testRow = new ArrayList<String>(Arrays.asList("a, b, c"));
+        ExportUtil.writeLines(new FileWriter(actual), testRow);
+        assertTrue(FileUtil.areFilesIdentical(expected, actual));
+    }
+
+    @Test
+    public void writeLines_greaterRows_returnsFalse() throws IOException {
```
Comment (iamjackslayer) [inline src/test/java/seedu/address/model/ExportUtilTest.java]: did you miss a test for fewer rows?

**Pull request merged**